<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生评奖评优系统</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menun">
    <property name="title">
     <string>学生信息</string>
    </property>
    <addaction name="actSearchPerson"/>
    <addaction name="actResult"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>部门管理</string>
    </property>
    <addaction name="actAdd"/>
    <addaction name="actClass"/>
    <addaction name="actZW"/>
    <addaction name="actCJImport"/>
    <addaction name="actAudit"/>
    <addaction name="actAuditExcel"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>评奖评优</string>
    </property>
    <addaction name="actExcel"/>
    <addaction name="actAddScore"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>系统管理</string>
    </property>
    <addaction name="actManage"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actDetail"/>
    <addaction name="actAbout"/>
    <addaction name="actExit"/>
   </widget>
   <addaction name="menun"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
   <addaction name="menu_5"/>
  </widget>
  <action name="actSearchPerson">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>信息查询</string>
   </property>
  </action>
  <action name="actManage">
   <property name="text">
    <string>账户管理</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="actDetail">
   <property name="text">
    <string>评奖评优规则设定</string>
   </property>
  </action>
  <action name="actClass">
   <property name="text">
    <string>班级管理</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actAdd">
   <property name="text">
    <string>学生信息编辑</string>
   </property>
  </action>
  <action name="actExcel">
   <property name="text">
    <string>评优申请</string>
   </property>
  </action>
  <action name="actResult">
   <property name="text">
    <string>评奖评优申请情况</string>
   </property>
  </action>
  <action name="actCJImport">
   <property name="text">
    <string>成绩录入</string>
   </property>
  </action>
  <action name="actAudit">
   <property name="text">
    <string>加分审核</string>
   </property>
  </action>
  <action name="actAddScore">
   <property name="text">
    <string>加分申请</string>
   </property>
  </action>
  <action name="actZW">
   <property name="text">
    <string>职务管理</string>
   </property>
  </action>
  <action name="actAuditExcel">
   <property name="text">
    <string>评优审核</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
